main.go

```diff
@@ -273,9 +276,12 @@ func (cmd *InstallCmd) Run(ctx *Context) error {
 			}
 			fmt.Printf("DONE\n")
 
+			fmt.Printf(" Master is %s\n", system.Master)
 			for rabbit := range system.Rabbits {
+				fmt.Printf(" Clients of rabbit %s\n", rabbit)
 
 				for _, compute := range system.Rabbits[rabbit] {
+					fmt.Printf(" checking... %s\n", compute)
```

Comment:
I know this is nit picking, but I've generally only use ellipses when running commands and there is continuation with the next message. Also, all messages should read like a sentence `"Checking for install on Compute Node %s"`